src/main/java/coordinate/Point.java

```diff
@@ -0,0 +1,69 @@
+package coordinate;
+
+import java.util.Objects;
+
+public class Point {
+    private final int x;
+    private final int y;
+
+    private Point(int x, int y) {
+        this.x = x;
+        if (x < 0 || x > 24) {
+            throw new IllegalArgumentException();
+        }
+
+        this.y = y;
+        if (y < 0 || y > 24) {
+            throw new IllegalArgumentException();
+        }
```

Comment:
이렇게 선언하는 방식이 가독성이 좋을 것 같은데 어떻게 생각하시나요? ☺️

```suggestion

        if (x < 0 || x > 24) {
            throw new IllegalArgumentException();
        }


        if (y < 0 || y > 24) {
            throw new IllegalArgumentException();
        }
        
        this.x = x;
        this.y = y;
```